charts/substra-backend/templates/statefulset-builder.yaml

```diff
@@ -137,48 +180,6 @@ spec:
           {{ end }}
         resources:
           {{- toYaml .Values.builder.resources | nindent 12 }}
-      initContainers:
-      {{- if .Values.privateCa.enabled }}
-      - name: add-cert
-        image: {{ include "common.images.name" .Values.privateCa.image }}
-        imagePullPolicy: {{ .Values.privateCa.image.pullPolicy }}
-        securityContext:
-          runAsUser: 0
-        command: ['sh', '-c']
-        args:
-        - |
-          {{- if .Values.privateCa.image.apkAdd }}
-          apt update
-          apt install -y ca-certificates openssl
-          {{- end }}
-          update-ca-certificates && cp /etc/ssl/certs/* /tmp/certs/
-        volumeMounts:
-          - mountPath: /usr/local/share/ca-certificates/{{ .Values.privateCa.configMap.fileName }}
-            name: private-ca
-            subPath: {{ .Values.privateCa.configMap.fileName }}
-          - mountPath: /tmp/certs/
-            name: ssl-certs
-      {{- end }}
-      - name: wait-minio
-        image: jwilder/dockerize:0.6.1
-        command: ['dockerize', '-wait', 'tcp://{{ .Release.Name }}-minio:9000']
-      {{- if .Values.kaniko.cache.warmer.cachedImages }}
-      - name: kaniko-cache-warmer
-        image: {{ include "common.images.name" .Values.kaniko.cache.warmer.image }}
-        args:
-        - "--cache-dir=/cache"
-        {{- range .Values.kaniko.cache.warmer.cachedImages }}
-        - "--image={{ . }}"
-        {{- end }}
-        - "--verbosity=debug"
-        volumeMounts:
-          - name: docker-cache
-            mountPath: /cache
-            readOnly: False
-          {{- if .Values.kaniko.dockerConfigSecretName }}
-          - name: docker-config
-            mountPath: /kaniko/.docker
-          {{- end }}
       {{- end}}
```

Comment:
Possibly orphaned `{{- end}}` ?

Reply:
Indeed I think that `{{ -end }}` was misplaced. It should belong with this [if](https://github.com/Substra/substra-backend/blob/chore/wait-pgsql-builder/charts/substra-backend/templates/statefulset-builder.yaml#L80).
```yaml
      {{- if .Values.kaniko.cache.warmer.cachedImages }}
```

Nicely spotted, thank you.